bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractOptionsFieldImpl.java

```diff
@@ -95,15 +100,23 @@ public Object[] getEnums() {
     }
 
     @Override
-    public String[] getEnumNames() {
+    public RichText[] getEnumNames() {
```

Comment:
You just can't change public API syntax, please mark this as deprecated and introduce a new API. This would break existing customer forms.